scripts/sysman/EXT_Exp_MemState.rst

```diff
@@ -0,0 +1,32 @@
+<%
```

Comment:
could you please rename this file as well to "EXT_Exp_MemPageOfflineState.rst"?

Reply:
Done.